_CodingChallenges/69.4-steering-evolution.md

```diff
@@ -31,6 +31,15 @@ parts:
   url: "/CodingChallenges/69.3-steering-evolution"
 - title: "Evolutionary Steering Behaviors - Part 5"
   url: "/CodingChallenges/69.5-steering-evolution"
+  
+  contributions:
+  - title: "My Evolutionary Steering Agents with predators"
+    author:
+      name: "Ben Ashworth"
+    url: "https://talkingchef.co.uk/codingTrain/genEco/"
+    
+    
+---
```

Comment:
This line is duplicated, please remove it.

Reply:
This is done now